Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(simXwithRC.py:3839241): Gdk-CRITICAL **: 08:48:24.437: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
